sopel/cli/run.py

```diff
@@ -30,11 +30,9 @@
     tools.stderr('Error: Sopel requires Python 2.7+ or 3.3+.')
     sys.exit(1)
 if sys.version_info.major == 2:
-    now = time.time()
-    state = 'has reached end of life'
-    if now >= 1588291200:  # 2020-05-01 00:00:00 UTC
-        state += ' and will receive no further updates'
-    tools.stderr('Warning: Python 2.x %s. Sopel 8.0 will drop support for it.' % state)
+    tools.stderr(
+        'Warning: Python 2.x has reached end of life and will receive '
+        'no further updates. Sopel 8.0 will drop support for it.')
```

Comment:
brace style nitpick
```python
        'no further updates. Sopel 8.0 will drop support for it.'
    )
```
I also prefer double quotes >.>

Reply:
NAK for brace style here; it's quite clear this way with only one arg. For calls with multiple args, I'd definitely do it your way.

As for double quotes… meh. I just kept what was already here, and I don't feel like putting this through another two-hour Travis queue just for that.